flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/translators/SinkTransformationTranslator.java

```diff
@@ -357,7 +357,9 @@ private <I, R> R adjustTransformations(
                     // In this case, the subTransformation does not contain any customized
                     // parallelism value and will therefore inherit the parallelism value
                     // from the sinkTransformation.
-                    subTransformation.setParallelism(transformation.getParallelism());
+                    subTransformation.setParallelism(
+                            transformation.getParallelism(),
+                            transformation.isParallelismConfigured());
```

Comment:
I haven't followed the runtime development for a while but is `isParallelismConfigured` also applicable for streaming jobs or only batch?

Reply:
The `isParallelismConfigured` flag applies to both streaming and batch jobs in Flink. However, for streaming jobs, it currently does not trigger any specific behavior or actions.